File: python/Film.py
class Film:
    # Constructor untuk membuat objek Film
    def __init__(self, id_film, judul, genre, durasi):
        # Atribut private menggunakan underscore
        self._id = id_film
        # Menyimpan judul film
        self._judul = judul
        # Menyimpan genre film
        self._genre = genre
        # Menyimpan durasi film dalam menit
        self._durasi = durasi

    # Getter dan Setter ID Film
    def getId(self):
        return self._id
    def setId(self, id_film):
        self._id = str(id_film)

    # Getter dan Setter Judul
    def getJudul(self):
        return self._judul
    def setJudul(self, judul):
        self._judul = judul

    # Getter dan Setter Genre
    def getGenre(self):
        return self._genre
    def setGenre(self, genre):
        self._genre = genre

    # Getter dan Setter Durasi
    def getDurasi(self):
        return self._durasi
    def setDurasi(self, durasi):
        self._durasi = durasi

    # Method untuk menampilkan informasi film
    def tampilkan_info(self):
        print(f"ID Film     : {self._id}\n" f"Judul       : {self._judul}\n" f"Genre       : {self._genre}\n" f"Durasi      : {self._durasi} menit"
        )

File: php/Film.php
<?php

// Membuat class Film
class Film{
    // Atribut private untuk poster film
    private $poster_film;
    // Atribut private untuk ID film
    private $id;
    // Atribut private untuk judul film
    private $judul;
    // Atribut private untuk genre film
    private $genre;
    // Atribut private untuk durasi film
    private $durasi;

    // Constructor Film
    public function __construct($poster_film, $id, $judul, $genre, $durasi){
        // Menyimpan poster film
        $this->poster_film = $poster_film;
        // Menyimpan ID film
        $this->id = $id;
        // Menyimpan judul film
        $this->judul = $judul;
        // Menyimpan genre film
        $this->genre = $genre;
        // Menyimpan durasi film
        $this->durasi = $durasi;
    }

    // Getter dan Setter poster film
    public function getPosterFilm(){
        return $this->poster_film;
    }
    public function setPosterFilm($poster_film){
        $this->poster_film = $poster_film;
    }

    // Getter ID film
    public function getId(){
        return $this->id;
    }
    // Setter ID film
    public function setId($id){
        $this->id = $id;
    }

    // Getter dan Setter Judul
    public function getJudul(){
        return $this->judul;
    }
    public function setJudul($judul){
        $this->judul = $judul;
    }

    // Getter dan Setter Genre
    public function getGenre(){
        return $this->genre;
    }
    public function setGenre($genre){
        $this->genre = $genre;
    }

    // Getter dan Setter Durasi
    public function getDurasi(){
        return $this->durasi;
    }
    public function setDurasi($durasi){
        $this->durasi = $durasi;
    }

    // Method menampilkan informasi film
    public function tampilkanInfo(){
        echo "ID Film : ".$this->id."<br>";
        echo "Judul : ".$this->judul."<br>";
        echo "Genre : ".$this->genre."<br>";
        echo "Durasi : ".$this->durasi." menit<br>";
    }
}
?>
